library/init/data/nat/lemmas.lean

```diff
@@ -119,17 +114,13 @@ instance : comm_semiring nat :=
  add_assoc      := nat.add_assoc,
  zero           := nat.zero,
  zero_add       := nat.zero_add,
- add_zero       := nat.add_zero,
```

Comment:
I think you can delete theorems `nat.add_zero` and similar that were removed from the structures now.

Reply:
I already deleted the ones that weren't used in the proofs of other properties.